### PR TITLE
Added rows to table dynamically, Indentation Correction  and Variable name change is done.

### PR DESCRIPTION
**Pull request by Harshwardhanwtagline**

### CODE REVIEW

File: table.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ 
+        
+        const arr = [{"name": "Harsh", "mail":"Hvwaghela@", "gender":"male", "hobbies": ["sports"], "country":"india", "state":"gujarat", "city":"surat"}, 
+        {"name": "Harry", "mail":"Harrywaghela@", "gender":"male", "hobbies": ["sports", "Reading"], "country":"india", "state":"gujarat", "city":"surat"}]
+        window.onload = function(){
+            for(let i of arr){
```
Comment (himanshu-suthar-tagline) [inline table.js]: Change name `i` to something meaningful.

Comment (Harshwardhanwtagline) [reply table.js]: Okay Sir

File: q2.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                    <option value="surat">Surat</option>
+                    <option value="ahmedabad">Ahmedabad</option>
+                    <option value="bharuch">Bharuch</option>
+                 </select></br></br>
+                <div style = "
+                        border-radius: 65%;
+                        border-background: black;
+                        display: flex;
+                        justify-content: space-evenly;">
+                        <button id= "button1" type="button">Submit</button>
```
Comment (himanshu-suthar-tagline) [inline q2.html]: Format the document properly. I can see here there is no space before `=` in `id= `. Please either give space to right and left both sides either don't give.

Comment (Harshwardhanwtagline) [reply q2.html]: OKay Sir

File: q2.html [hunk 3]
```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
```
Comment (himanshu-suthar-tagline) [inline q2.html]: Change filename `q1.html` to `index.html`

File: table.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+                var table = document.getElementById("myTable");
```
Comment (himanshu-suthar-tagline) [inline table.js]: Use `let` instead of `var`

File: q2.html (hunk 3 above)
Comment (Harshwardhanwtagline) [reply q2.html]: Okay Sir

File: table.js (hunk 4 above)
Comment (Harshwardhanwtagline) [reply table.js]: Okay Sir I will Change it.

**Comment by himanshu-suthar-tagline** [comment]
Also, you don't have to name the PR as `Code Review of Q2`. You just have to write what you did in this commit. For example, added rows to table dynamically.

**Comment by himanshu-suthar-tagline** [comment]
The commit message should be proper. It should not be like `Q2 completed`. It should shows what you did in this commit in brief. For example, here you added the javascript to add the rows to the table then your commit message can be like `Added javascript logic to append rows to table`. Please be aware from the next commit message.

**Pull request merged**